lib/puppet/ssl/host.rb

```diff
@@ -484,6 +478,30 @@ def load_certificate_bundle(bundle_string)
     end
   end
 
+  # Fetches the crl bundle from the CA server; because this is used for the first
+  # fetch of the crl, we need to pass in a certificate store that has no crls
+  # attached to it, but still utilizes the localcacert.
+  # @raise [Puppet::Error] if response from the server is not a valid certificate
+  #                        bundle
```

Comment:
> not a valid certificate bundle

?

Reply:
Should probably be "PEM-encoded CRL chain".